### DISCUSSION TITLE
一键部署这里似乎有点问题，没有任何应用可选。

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#6846] [Q&A]
我在应用商店按装了好几个应用，但是创建网站的时候在一键部署页面一个都不能选。我记得在创建第一个网站的时候是可以选择的，但是当我创建了一个本地的应用并使用反向代理那个选项之后，这里就完全没有逻辑了，现在我安装哪个应用好像都没法选择一键部署。这是在我删除了之前的哪个本地应用之后，否则这里指挥显示那个本地应用。不知道这里一键部署在什么情况下可以选择应用。

**Comment by wanghe-fit2cloud** [comment]
> 不知道这里一键部署在什么情况下可以选择应用。

支持应用商店中 `建站` 标签的应用。
